static/src/js/components/SearchSidebar/SearchSidebarHeader.js

```diff
@@ -1,16 +1,158 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Link
+// withRouter
+// Route,
+// // Routes,
+// Switch
+} from 'react-router-dom'
+
+import { parse } from 'qs'
+import { FaTimes } from 'react-icons/fa'
+
+// import PortalLinkContainer from '../../containers/PortalLinkContainer/PortalLinkContainer'
 
 import SearchFormContainer from '../../containers/SearchFormContainer/SearchFormContainer'
+import Button from '../Button/Button'
+import { locationPropType } from '../../util/propTypes/location'
+import { stringify } from '../../util/url/url'
 
 import './SearchSidebarHeader.scss'
 
+export const mapStateToProps = (state) => ({
```

Comment:
components should not talk to redux, containers pass any props derived from redux down into the components